client/src/config/clientConfig.ts

```diff
@@ -8,28 +8,25 @@ import { globalCacheUpdates, ConflictLogger } from '../helpers';
 import { getAuthHeader } from '../auth/keycloakAuth';
 import { WebNetworkStatus } from './WebNetworkStatus';
 
-const httpUri = process.env.REACT_APP_SERVER_URL || 'http://localhost:4000/graphql';
-const httpsEnabled = httpUri.startsWith('https://')
-const httpEnabled = httpUri.startsWith('http://')
+let httpUri = 'http://localhost:4000/graphql';
+let wsUri = 'ws://localhost:4000/graphql';
 
-if (!httpEnabled && !httpsEnabled) {
-  throw new Error(`invalid server url ${httpUri} must begin with https:// or http://`)
+if (process.env.REACT_APP_URI_FORMAT === 'RELATIVEURI') {
```

Comment:
I went with this as we can have also the explicit format in the future. Actually let me add that as supported option as well. WDYT? Any value in having support for providing exact url?